DevEdu.API/DevEdu.Business.Tests/Data/CourseData.cs

```diff
@@ -0,0 +1,87 @@
+﻿using DevEdu.DAL.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DevEdu.Business.Tests
+{
+    public class CourseData
+    {
+        public CourseData()
+        {
+
+        }
+        /// <summary> This method returns CourseTopicDto, you can choose 1-3 version.</summary>
+        public static CourseTopicDto GetCourseTopicDto(int vesionOfDto)
+        {
+            CourseTopicDto courseTopicDto;
+            switch (vesionOfDto)
```

Comment:
why do we need a switch here?
the only usage of this method I found: CourseData.GetCourseTopicDto(1)